src/functional/program_processors/runners/gtfn_cpu.py

```diff
@@ -82,3 +83,6 @@ def __name__(self) -> str:
 
 
 run_gtfn: Final[ppi.ProgramProcessor[None, ppi.ProgramExecutor]] = GTFNExecutor(name="run_gtfn")
+run_gtfn_no_transforms: Final[ppi.ProgramProcessor[None, ppi.ProgramExecutor]] = GTFNExecutor(
```

Comment:
We decided to move this part to the tests, and annotate the apply_transforms flag linking to an issue with the desired behavior as outlined above.